.circleci/config.yml

```diff
@@ -150,6 +168,16 @@ jobs:
       - slack/status:
           fail_only: true
           only_for_branches: develop
+  testNode14Windows:
+    executor:
+      name: win/default
+      size: "large"
+      shell: bash.exe
+    steps:
+      - checkout
+      - setup_windows
+      - runtests:
+          enableIntegrationTests: false
```

Comment:
Before we're running the integration tests on the master branches on Windows:
https://github.com/mobify/mobify-platform-sdks/blob/salesforce-develop/appveyor.yml#L59-L62

Now we're not running any integration test on Windows. Is that intentional?

Reply:
Good catch, I must has copied the `enableIntegrationTests: false` parameter from another job. I'll remove that so that the integration tests are run on those branches since `enableIntegrationTests` defaults to `true`.